src/Native/System.Security.Cryptography.Native/pal_eckey.cpp

```diff
@@ -12,33 +12,59 @@ extern "C" void CryptoNative_EcKeyDestroy(EC_KEY* r)
     EC_KEY_free(r);
 }
 
-extern "C" EC_KEY* CryptoNative_EcKeyCreateByCurveName(int32_t nid)
```

Comment:
I think we probably want to leave the existing functions for compatibility with RC2.  If we've removed the calls by RTM we can remove the functions in 1.1.